scripts/get

```diff
@@ -65,11 +65,15 @@ verifySupported() {
 # checkLatestVersion checks the latest available version.
 checkLatestVersion() {
   # Use the GitHub API to find the latest version for this project.
-  local latest_url="https://api.github.com/repos/kubernetes/helm/releases/latest"
+  local latest_url="https://github.com/kubernetes/helm/releases/latest"
   if type "curl" > /dev/null; then
-    TAG=$(curl -s $latest_url | awk '/\"tag_name\":/{gsub( /[,\"]/,"", $2); print $2}')
+    TAG=$(curl -SsL $latest_url | awk '/\/tag\//' | head -n 1 | cut -d '"' -f 2 | awk '{n=split($NF,a,"/");print a[n]}')
   elif type "wget" > /dev/null; then
-    TAG=$(wget -q -O - $latest_url | awk '/\"tag_name\":/{gsub( /[,\"]/,"", $2); print $2}')
+    TAG=$(wget -q -O - $latest_url | awk '/\/tag\//' | head -n 1 | cut -d '"' -f 2 | awk '{n=split($NF,a,"/");print a[n]}')
+  fi
+  if [ "x$TAG" == "x" ]; then
```

Comment:
Rather than explaining it myself, I think this post in StackOverflow best defines what I want to achieve:
http://stackoverflow.com/questions/174119/why-do-shell-script-comparisons-often-use-xvar-xyes/174331#174331

I consider it a good practice, although it does not matter in this case.
If you prefer to use `-z` I will update the PR